packages/react/src/components/Storage/FileUploader/types.ts

```diff
@@ -39,6 +39,7 @@ export interface PreviewerProps {
   isLoading: boolean;
   isSuccessful: boolean;
   hasMaxFilesError: boolean;
+  maxFiles: number;
```

Comment:
`maxFiles` is not required on `FileUploaderProps`, have we tested edge cases? What happens if it's not specified by customers?

Reply:
Thanks for confirming!

Reply:
Is there any S3 limit to how many files can be uploaded at a time? I assume we handle that gracefully?

Reply:
It seems s3 should scale with load. So you can do up to 3500 PUTs per second according to this [article](https://docs.aws.amazon.com/AmazonS3/latest/userguide/optimizing-performance.html). But you could perhaps scale that more. 

I want to say here the bottle neck would be the browser, and the OS for the user.